lifetimes/utils.py

```diff
@@ -114,6 +115,32 @@ def to_period(d):
     return customers[['frequency', 'recency', 'T']].astype(float)
 
 
+def calculate_alive_path(model, transactions, datetime_col, units, freq='D'):
```

Comment:
In other lifetimes functions we use `t` to denote number of time units. Mind changing it to `t`?